test/.setup.js

```diff
@@ -1,20 +1,3 @@
 require('babel-register')();
 
-var jsdom = require('jsdom').jsdom;
-
-var exposedProperties = ['window', 'navigator', 'document'];
-
-global.document = jsdom('');
-global.window = document.defaultView;
-Object.keys(document.defaultView).forEach((property) => {
-  if (typeof global[property] === 'undefined') {
-    exposedProperties.push(property);
-    global[property] = document.defaultView[property];
-  }
-});
-
-global.navigator = {
-  userAgent: 'node.js'
-};
-
-documentRef = document;
```

Comment:
This variable did not know whether it is referenced from anywhere. Do you know whether or not it is necessary during setup?